src/modules/admin/components/services/ServiceCategoryTable.tsx

```diff
@@ -50,7 +50,6 @@ const ServiceCategoryTable = () => {
         columns={columns}
         pagePath='serviceCategories'
         noData='No service categories'
-        showFilters
```

Comment:
I removed this because `ServiceCategoryFilterOptions` doesn't exist, so no filters were rendering anyway pre-refactor. Is this a bug? If so I'll create a ticket, but I figured it's probably just a copy/paste error.

Reply:
Hm looks like this controls whether the top toolbar shows up at all. Thats not great naming.

With `showFilters`:
<img width="1379" alt="Screenshot 2024-04-26 at 12 36 12 PM" src="https://github.com/greenriver/hmis-frontend/assets/5333982/eda91301-be5b-4281-a384-b31221e36864">


Without `showFilters`:
<img width="1386" alt="Screenshot 2024-04-26 at 12 36 18 PM" src="https://github.com/greenriver/hmis-frontend/assets/5333982/68c2115b-f251-4280-b202-41fd2dffb3c6">

Reply:
Ah I see, I think I'd like to keep it and rename it `showTopToolbar`. Looks like it is also used in other places that have no filters (the ones mentioned in my other comment [below](https://github.com/greenriver/hmis-frontend/pull/749#discussion_r1580173837))

Reply:
great 👍